test/e2e/tests/add-hide-token.spec.js

```diff
@@ -55,6 +55,7 @@ describe('Hide token', function () {
   });
 });
 
+/* eslint-disable-next-line mocha/max-top-level-suites */
```

Comment:
With the updates to the ESLint config, these test files are now failing the lint step. I suspect this is the case because not all of the Mocha rules were getting applied to the e2e tests previously and now they are